src/service/repo/anonymousMessageRepo.ts

```diff
@@ -30,8 +30,8 @@ interface AnonymousMessageRepoTypes {
     attachments: any,
     replyMessageId?: string
   ) => Promise<any>;
-  getAllAnonymousChannels: () => Promise<ChannelData[]>;
-  getAllAnonymousPostNotifications: () => Promise<AnonymousPostNotification[]>;
+  getAllAnonymousChannels: (timeStamp: string) => Promise<ChannelData[]>;
+  getAllAnonymousPostNotifications: (timeStamp: string) => Promise<AnonymousPostNotification[]>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `timeStamp` parameter to the `getAllAnonymousChannels` and `getAllAnonymousPostNotifications` functions is a good improvement. It allows for more efficient data fetching by only retrieving new data since the last fetch.



<!-- This is an auto-generated comment by OSS CodeRabbit -->